resources/views/device/header.blade.php

```diff
@@ -0,0 +1,24 @@
+<div class="panel panel-default">
+    <div class="panel-body {{ $alert_class }}">
+        <img src="{{ url($device->logo()) }}" title="{{ $device->logo() }}" class="device-icon-header pull-left" style="max-height: 100px">
+        <div class="pull-left" style="margin-top: 5px;">
+            @if($parent_id)
+                <a href="{{ route('device', $parent_id) }}" title="@lang('VM Host')"><i class="fa fa-server fa-fw fa-lg"></i></a>
+            @endif
+            @if(\LibreNMS\Alert\AlertUtil::isMaintenance($device_id))
+                <span title="@lang('Scheduled Maintenance')" class="fa fa-wrench fa-fw fa-lg"></span>
+            @endif
+            <span style="font-size: 20px;">{!! \LibreNMS\Util\Url::deviceLink($device) !!}</span><br/>
```

Comment:
Use our fancy blade extensions :)
```suggestion
            <span style="font-size: 20px;">@deviceLink($device)</span><br/>
```